src/content/en/updates/2019/07/image-support-for-async-clipboard.md

```diff
@@ -112,37 +116,40 @@ permissionStatus.onchange = () => {
 };
 ```
 
-## 🆕 The new image-focused portion of the Asynchronous Clipboard API {: #images }
+## The new image-focused portion of the Asynchronous Clipboard API {: #images }
 
-### Copy: Writing an image to the clipboard {: #copy-image }
+### Copy: writing an image to the clipboard {: #copy-image }
 
 The new `navigator.clipboard.write()` method can be used for copying images
 to the clipboard. Like [`writeText()`](#copy-text), it is asynchronous, and
 Promise-based. Actually, `writeText()` is just a convenience method for the
 generic `write()` method.
 
-In order to write an image to the clipboard, you need the image as a
-[`Blob`](https://developer.mozilla.org/en-US/docs/Web/API/Blob). One way to
-achieve this is by `fetch`ing (or `XMLHttpRequest`ing) the image from a
-server and getting the response body as a [Blob][blob] (or for XHR,
-by setting the [`responseType`][blog-response-type] to `'blob'`).
-Another method to `Blob`ify an image is to write the image to a canvas, then
-call the `canvas`’s [`toBlob()`][to-blob] method.
+To write an image to the clipboard, you need the image as a
+[`Blob`][blob]. One way to do
+this is by requesting the image from an server by calling `fetch()` (or
+`XMLHTTPReuest()`). The `response` object returned by `fetch()` has a [`blob()`
+method][blob-method] and
+`XMLHTTPRequest()` let's you set `"blob"` as the [`responseType`][blog-response-type]. 
+
+Calling the server may not be desireable or possible for a variety of reasons.
+Fortunately, you can also write the image to a canvas, then call
+[`HTMLCanvasElement.toBlog()`][to-blob].
 
-[blob]: https://developer.mozilla.org/en-US/docs/Web/API/Body/blob
+[blob]: https://developer.mozilla.org/en-US/docs/Web/API/blob
+[blob-method]: https://developer.mozilla.org/en-US/docs/Web/API/Body/blob
 [to-blob]: https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toBlob
 [blog-response-type]: https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/responseType#Value
 
-Next, pass an array of `ClipboardItem`s as a parameter to the `write()` method.
+Next, pass an array of `ClipboardItem` objects as a parameter to the `write()` method.
 Currently you can only pass one image at a time, but we plan to add support for
 multiple images in the future.
 
 The `ClipboardItem` takes an object with the MIME type of the image as the key,
```

Comment:
I don't understand this at all. I tried to find MIME types in the creation of Clipboard instances in the spec and couldn't.

Reply:
This refers to this step in the code:

```js
await navigator.clipboard.write([
    new ClipboardItem(Object.defineProperty({}, blob.type, {
      value: blob,
      enumerable: true
    }))
  ]);
```

Me, too(?), I think the API is a bit wordy here, but I believe it's like that because eventually it will be opened to all MIME types, meaning it needs full flexibility. I can imagine there will be convenience methods introduced in the future for more common clipboard tasks like fiddling with images (text already has a convenience method, as described in the article).

Reply:
I still don't see a MIME type.

Reply:
It's the `blob.type` part in the code.

Reply:
@tomayac I've added a line below that I think clarifies the matter. Please take a look.